src/main/java/com/shippo/model/Pickup.java

```diff
@@ -30,6 +30,7 @@ public class Pickup extends APIResource {
 	Object timezone;
 	Object messages;
 	Object metadata;
+	// TODO: add private boolean test attribute.  Today response payload does not include test attribute.
```

Comment:
are the docs inaccurate? It says an `is_test` attribute exists https://goshippo.com/docs/reference#pickups

Reply:
mapped `is_test` in payload to `test` in model